__tests__/react.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+import Emote from "../client/components/Emote";
+// import Navbar from "../client/components/Navbar"; To be tested one day, when buttons have functionality
+// import App from "../client/App"; To be tested one day, gives errors for now
+
+describe("Unit testing React components", () => {
+  describe("Emote", () => {
+    let post;
+    const props = {
+      msg: "&#128065 &#128068 &#128065",
+    };
+
+    beforeAll(() => {
```

Comment:
beforeAll works if there's one single test, but I think using beforeEach is better style because it'll do the same and also allow us to add more tests to this describe block without a second thought.